MapboxSearch/place-autocomplete/src/main/java/com/mapbox/search/autocomplete/PlaceAutocomplete.kt

```diff
@@ -69,16 +70,20 @@ public interface PlaceAutocomplete {
          * Note that this class requires [Manifest.permission.ACCESS_COARSE_LOCATION] or
          * [Manifest.permission.ACCESS_FINE_LOCATION] to work properly.
          *
+         * @param apiType to be used. Default is [CoreApiType.SBS]
+         *
          * @return a new instance instance of [PlaceAutocomplete].
          */
         @JvmStatic
         @JvmOverloads
         public fun create(
             locationProvider: LocationProvider? = defaultLocationProvider(),
+            apiType: CoreApiType = CoreApiType.SBS,
```

Comment:
Should this be `CoreApiType.SearchBox`?

Reply:
This would incur a change to the public API but seems to be a non-breaking one. I fixed that.